### PR TITLE
Issue 99 - Collapsible filter/legend

### PR DESCRIPTION
**Pull request by amaxama**
### What
Add collapsability to top filters and toggle legend on map when filters are collapsed

### Why
Issue #99 

### Ensure the following interactions work as expected. Please test using a mobile form factor.
- [x] Tapping on a marker on the map displays information about the marker in a popup.
- [x] Tapping the "Show list of locations" button replaces the map view with a list view.
- [x] Tapping an item in the list replaces the list view with a map view, and navigates the map to the tapped item on the map.
- [x] Tapping one of the ✅'s in the legend filters items on the map and in the list.
- [x] Changing the language to spanish changes things on the page.
- [x] Clicking the Help/Info button opens and closes a menu with information.
- [x] Clicking the X Close button on the Help/Info screen closes the modal.

### Check the app in the following web browsers:
- [x] Chrome
- [ ] Firefox
- [ ] Edge
- [x] Safari

### CODE REVIEW

File: src/styles/theme.css [hunk 1]
```diff
@@ -20,6 +20,7 @@
     --primary-700: hsl(var(--primary), 35%);
     --primary-800: hsl(var(--primary), 25%);
     --primary-900: hsl(var(--primary), 15%);
+    --primary-blue: hsl(var(--primary), 10%);
```
Comment (amaxama) [inline src/styles/theme.css:23]: This is the original dark blue used for the title header - open to a better variable name if there are suggestions!

File: public/index.html [hunk 2]
```diff
@@ -48,18 +48,38 @@
...
+              <img class="refresh-image" alt="${lang_name}" src="/images/refresh.png">
+              <span class="refresh-text" data-translation-id="refresh">
+                Refresh
+              </span>
+            </button>
+            <div class='counter'>
+              <script type="text/javascript" src="//counter.websiteout.net/js/7/0/41000/0" ></script>
+            </div>
+          </div>
+        </div>
```
Comment (amaxama) [inline public/index.html:82]: Most of this is just indentation changes

File: src/styles/styles.css [hunk 3]
```diff
@@ -133,35 +137,17 @@ input[type='checkbox'] + label {
 
 .white-and-blue {
   color: white;
-  background-color: rgba(0,0,50,.9);
+  background-color: var(--primary-blue);
 }
 
 /* component: title */
 .title {
-  padding: 10px 15px 0px 15px;
```
Comment (amaxama) [inline src/styles/styles.css:141]: Moved all legend and filter related items to `filter.css`

File: src/styles/styles.css [hunk 4]
```diff
@@ -52,7 +52,7 @@ a.txt-white:hover {
 
 .content {
   z-index: 1;
-  width: 33.3333%;
+  width: 36%;
```
Comment (amaxama) [inline src/styles/styles.css:55]: Mad just a little wider so title fits on one line more often

File: src/i18n/amh.json [hunk 5]
```diff
@@ -22,6 +22,10 @@
   "filter_by_both": "ለ ሁለቱም ክፍት ነው ",
   "filter_by_closed": "ለጊዜው ዝግ ነው ",
   "filter_by_unknown": "ሁኔታው አይታወቅም ",
+  "receiving": "receiving",
```
Comment (amaxama) [inline src/i18n/amh.json:25]: All of these are just placeholders for now!

**Comment by amaxama** [comment]
Here's an example of what it looks like (animation is slow cause my computer is laggy, it should be a bit more smooth than that): 
filter-legend-toggle

**Comment by mc-funk** [comment]
So exciting! 
I’m wondering if there might be.a better placement for the toggle caret — on mobile for me I am struggling to tap the caret Instead of the checkbox it’s next to. Maybe it would be worth nudging down the font of TCMAP header and getting them on the same line?

**Comment by amaxama** [comment]
@mc-funk Yeah... I played around with a few places to put it..I had originally thought of putting it in line with the title but I actually kind of want to put a kebab menu there for overflow actions like "refresh"....I could try putting it over to the right though and see how that goes. Regardless, I just tried it on mobile and the touch area that picks up tap input is definitely not big enough so I'm going to increase that for sure. I'll try that first and see if that makes a difference for you and then I can try it over to the right if it's sill not great

**Pull request closed without merging**